sympy/printing/latex.py

```diff
@@ -166,6 +166,7 @@ class LatexPrinter(Printer):
         "parenthesize_super": True,
         "min": None,
         "max": None,
+        "mat_compact": True,
```

Comment:
I still don't think `mat_compact` is the right kind of option to add. Rather there should be a general option for shortening all possible outputs (not just matrices). Its purpose should be to avoid producing absurdly large output in general even if for now it is only applied to matrices.

Reply:
I agree, just forgot to change mat_compact to truncate in a few spots.  The docstring specifies that truncate currently only truncates matrix but might be extended to truncate other things in the future.